inventory/byo/hosts.origin.example

```diff
@@ -113,6 +113,9 @@ openshift_release=v3.6
 # Downgrades are not supported and will error out. Be careful when upgrading docker from < 1.10 to > 1.10.
 # docker_version="1.12.1"
 
+# Specify whether to --enable-selinux on Docker (set false if using overlay/2 storage drivers before RHEL 7.4)
```

Comment:
nit: May be worth noting that it will change ``container-daemon.json`` on container-engine. Eventually everything will move to a json file:

```
    "selinux-enabled": {{ l_docker_selinux_enabled | lower }},
```